prov/usnic/src/usdf_cq.c

```diff
@@ -428,21 +454,24 @@ usdf_cq_post_soft(struct usdf_cq_hard *hcq, void *context, size_t len,
 }
 
 void
-usdf_cq_post_soft_context(struct usdf_cq_hard *hcq, void *context, size_t len)
+usdf_cq_post_soft_context(struct usdf_cq_hard *hcq, void *context,
```

Comment:
All of these `usdf_cq_post_soft_${TYPE}` functions should go away and all users of them should be replaced with a direct call to `usdf_cq_post_soft`.  `usdf_cq_post_soft` no longer depends on the value of `format`.